responses/responses_th.json

```diff
@@ -177,5 +177,27 @@
     "list_of_words": ["หนู", "เธอ", "อายุ", "เท่า", "ไหร่"],
     "is_single_response": false,
     "required_word": []
+  },
+
+  "lazy_th": {
+    "response": [
+      "หนูก็เหมือนกันค่ะ คุณพี่ควรไปพักนะคะ",
+      "ทางนี้ก็เช่นกันค่ะ คุณพี่ควรไปพักนะคะ"
+    ],
+    "list_of_words": ["เหนื่อย","เบื่อ","ขี้เกียจ"],
+    "is_single_response": false,
+    "required_word": []
+  },
+
+  "sleep_th": {
+    "response": [
+      "ไปนอนสิคะ คุณพี่ควรไปพักนะคะ",
+      "คุณพี่ควรไปนอนนะคะ",
+      "คุณพี่ควรไปนอนสิคะ"
```

Comment:
I think this should be: "คุณพี่ควรไปนอนได้แล้วนะคะ"

Reply:
Ok